data-raw/ccao_ids.R

```diff
@@ -153,7 +155,9 @@ right <- c(
   "claire",
   "jean",
   "damon",
-  "michael"
+  "michael",
+  "caroline",
+  "matt" # again
```

Comment:
**[Suggestion, non-blocking]** Maybe `matthew` to distinguish this name from the one above?